src/commands/node.mjs

```diff
@@ -460,6 +455,15 @@ export class NodeCommand extends BaseCommand {
               title: `Start node: ${chalk.yellow(nodeId)}`,
               task: async () => {
                 await self.k8.execContainer(podName, constants.ROOT_CONTAINER, ['rm', '-rf', `${constants.HEDERA_HAPI_PATH}/data/logs`])
+
+                // copy application.env file if required
+                if (ctx.config.applicationEnv) {
+                  const stagingDir = Templates.renderStagingDir(self.configManager, flags)
+                  const applicationEnvFile = path.join(stagingDir, 'application.env')
+                  fs.cpSync(ctx.config.applicationEnv, applicationEnvFile)
```

Comment:
Talking with Nathan it sounds like all of these config files need to be a ConfigMap that is mounted in.

Reply:
As discussed, we shall need to work on a separate PR where `application.env` will be mounted from a config-map to `/etc/network-node/` directory. 

This will also require updating the docker images in `full-stack-testing/docker` to ensure `application.env` is loaded from `/etc/network-node/`:
`docker/ubi8-init-java21/network-node.service`
`docker/ubi8-init-java21/Dockerfile`

Reply:
See https://github.com/hashgraph/solo/issues/110